### DISCUSSION TITLE
Is there a way to get the entire selection in --preview

### DISCUSSION THREAD

**Discussion by u9g** [junegunn/fzf#3217] [Q&A]
Hi, I'm making a help visualizer for nushell, but some of the commands have spaces like `ansi gradient ...` so in the help menu, they show up as `ansi gradient`, but then when I go to preview `{1}` I only get ansi, which is bad for getting more specific information about the command. I'm curious if there's a way to disable this splitting or get the whole selection that's highlighted. Here's my command so far:
```sh
help commands | get name | str join "\n" | fzf --preview-window 'up,60%,border-bottom,+{2}+3/3,~3' --preview "nu -c 'print {1}'"
```

see 
![image](https://user-images.githubusercontent.com/43508353/226967201-c83f6765-aac6-412e-9dcf-d931c902d969.png)

**Comment by junegunn** [comment]
Please read through the README page and man page.

https://github.com/junegunn/fzf/blob/master/man/man1/fzf.1#L508
